### ISSUE TITLE
Floating label + date input = incorrect layout

### ISSUE THREAD

**Issue by Leon99** [mdbootstrap/mdb-ui-kit#1377]
This is what it looks like:

![image](https://user-images.githubusercontent.com/6840381/63986203-4e9ad600-cb16-11e9-8ec8-18a0172d0b8f.png)
Code is essentially taken from the documentation on floating labels with only input type modified.
```html
<fieldset class="form-group">
    <label for="Data_Dob" class="bmd-label-floating">Date of Birthday</label>
    <input type="date" class="form-control" id="Data_Dob" name="Data.Dob">
    <span class="bmd-help">Date of birthday</span>
</fieldset>
```
Apologies for the absence of CodePen, floating didn't work there for some reason.

Any suggestions on a workaround appreciated.

**Comment by Valronn** [comment]
### Add this to CSS ###

`input[type=date]::-webkit-datetime-edit {
    color: transparent;
}`
`input[type=date]:focus::-webkit-datetime-edit {
    color: black !important;
}`


 Here is full **[solution](https://codepen.io/valronn/pen/MWgvpyp)**

**Comment by Leon99** [comment]
Thanks @Valronn, I had tried that trick and other similar from https://stackoverflow.com/questions/28686288/remove-default-text-placeholder-present-in-html5-input-element-of-type-date, but most of them are unstable (stop working at some point) or make the content disappear as well. The only solution that works as intended is https://stackoverflow.com/a/23683687/8342076. So yes, I've found a workaround, but it'd be great to have it working without a hack (or, at least, add it to the documentation?).
